openfe/protocols/openmm_rfe/equil_rfe_settings.py

```diff
@@ -112,6 +108,18 @@ class RelativeHybridTopologyProtocolSettings(Settings):
     class Config:
         arbitrary_types_allowed = True
 
+    protocol_repeats: int = 3
+    """
+    Number of protocol repeats to run.  Default 3
+    """
+
+    @validator('protocol_repeats')
+    def must_be_positive(cls, v):
+        if v <= 0:
+            errmsg = "protocol_repeats must be a positive value"
```

Comment:
try and always spit back the bad value, e.g "protocol_repeats must be a positive value, got '0'"